pkg/webui/styles/main.styl

```diff
@@ -12,9 +12,34 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700')
-@import url('https://fonts.googleapis.com/css?family=IBM+Plex+Mono')
-
+/* source-sans-pro-regular */
```

Comment:
I'd prefer to have font definitions outsourced to a `font.styl` and then referenced in `include.styl`.

Reply:
I agree, will do.

Reply:
@bafonins @kschiffer What do you guys think to setting the font weights as variables and then i'll go through an update them all? i'm thinking `400 = fw.n (font weight normal)` `600 = fw.sb (font weight semibold )` `700 = fw.b (font weight bold )`

Reply:
I think it would be good to do this, but maybe not in this PR.

Also regarding my other comment, the font-weight should not be set to `600`, but `400` actually.

Reply:
@kschiffer I have included 'fonts.styl' in the 'main.styl' as including it in 'include.styl' was causing webpack to incorrectly rewrite the url paths. Is this ok? If not can you give me some insight on why you think this is not working correctly?

Reply:
This is due to `includes.styl` [being included automatically](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/config/webpack.config.babel.js#L82) by every stylus file, so we have access to the mixins and variables from everywhere. 
But this also leads to webpack trying to resolve the `url()` imports relative to whichever stylus file being used. So, I think it's indeed better to include this into `main.styl` (which resolves correctly from `pkg/webui/styles`), and it's more of a general style definition anyway, unlike the mixins and variables, which are meta.

Reply:
@kschiffer Is there anything else that needs doing here?

Reply:
See my review.